src/derivatives/functional-derivatives.md

```diff
@@ -0,0 +1,404 @@
+| `mean`           | Mean across the temporal/4th dimension of the data               |
+| `std`            | Standard deviation across the temporal/4th dimension of the data |
+| `tsnr`           | Temporal SNR (`mean / std`)                                      |
+| `sfs`            | Signal fluctuation sensitivity                                   |
+| `alff`           | Amplitude low frequency fluctuations                             |
+| `falff`          | Fractional amplitude of low frequency fluctuations               |
+| `reho`           | Regional homogeneity (voxelwise only)                            |
+| `dcb`, `dcw`     | Voxelwise degree centrality binary and weighted                  |
+| `ecb`, `ecw`     | Voxelwise eigenvector centrality binary and weighted             |
+| `lfcdb`, `lfcdw` | Local functional connectivity density                            |
+| `vmhc`           | Voxel mirrored homotopic connectivity                            |
+
+### Sidecar JSON (`*_boldmap.json`/`*_cbvmap.json`)
+
+The following metadata JSON fields are valid for derivative maps:
+
+<!-- This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("derivatives.fmri.FunctionalMapFields") }}
+
+## Time series
+
+A time series is a chronologically ordered series of numeric values.
+Time series will generally be stored as tables, with a row of column headers
+indicating the name of the series.
+In the case where the time series varies spatially, the data SHOULD be stored in
+an appropriate file, such as a 4D NIfTI file, a `.func.gii` GIFTI file,
+or a `.dtseries.nii` CIFTI-2 file.
+
+All time series files MUST be accompanied by a data dictionary in JSON format,
+consistent with the format described in
+[Common principles](../common-principles.md#tabular-files), which describes
+metadata for each column name.
+In the case of spatial time series files, the notion of column name does not
+apply, so column-level metadata may be applied to the entire file.
+In addition, the following fields apply to the entire file in all cases:
+
+| Field name        | Definition                                                                                                                                                     |
+| :---------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| SamplingFrequency | REQUIRED. Sampling frequency (in Hz) of all columns in the file. Special value `"TR"` indicates one sample per volume of a corresponding BOLD series.          |
```

Comment:
I think the actual TR value should be saved in the metadata for distributing the derivative. Otherwise users will need to find the original dataset to get the actual TR for analysis.

Reply:
The BOLD timing information should be associated with the BOLD file, whether raw or derived. I'm not sure the advantage of duplicating it in `timeseries.tsv`. What is your use case?

Reply:
Extracting time series from ROIs from multiple datasets of different TR, and we potentially need to resample all the data to the same sampling rate for time series model (e.g. autoregressive model).

Reply:
Good point, as there may not be an associated BOLD files. But in that case, would you not just set `SamplingFrequency` to `1/RepetitionTime`? Along with `StartTime`, that's a complete definition.

We should consider the `VolumeTiming` case for BOLD. We could see if `VolumeTiming`'s definition can accommodate an expansion along the lines of "In the case of ROI time series, the VolumeTiming is the timing information for the source BOLD series." Or we could create a `SampleTiming` array that makes it explicit. Or we follow PET's lead with blood recordings and add a `time` column in seconds.

I'm a little inclined toward the `time` column, as that would be a reasonable addition to any `timeseries.tsv`, and cover even and unevenly-spaced time series.

It would probably be good to have an explicit rule where timeseries with no associated BOLD series and no `time` column SHOULD NOT set `"SamplingFrequency": "TR"`.

Reply:
`time` column is the clearest way to me to include this piece of information. I like this idea.